umm-spec-lib/src/cmr/umm_spec/umm_g/granule.clj

```diff
@@ -21,7 +41,7 @@
   [umm-g-json]
   (let [coll-ref (umm-g->CollectionRef umm-g-json)]
     (g/map->UmmGranule {:granule-ur (:GranuleUR umm-g-json)
-                        ; :data-provider-timestamps data-provider-timestamps
+                        :data-provider-timestamps (umm-g->DataProviderTimestamps umm-g-json)
                         :collection-ref coll-ref
```

Comment:
Collection ref was already done (CMR-5059).

Reply:
Yes, so is CMR-5058.